package.json

```diff
@@ -39,7 +38,9 @@
   "dependencies": {
     "@babel/runtime-corejs3": "^7.14.7",
     "@material-ui/icons": "4.9.1",
-    "audio-recorder-polyfill": "^0.4.1"
+    "audio-recorder-polyfill": "^0.4.1",
+    "@ant-design/icons": "^4.6.2",
+    "antd": "^4.16.9"
```

Comment:
This lib should not be depend on these libs. You need to add ability to pass custom component for message render as this lib do it for input.  You can pass as custom only button for play audio